framework/CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ Yii Framework 2 Change Log
 - Enh #15357: Added multi statement support for `yii\db\sqlite\Command` (sergeymakinen)
 - Enh #15360: Refactored `BaseConsole::updateProgress()` (developeruz)
 - Enh #15415: Added transaction/retry support for `yii\db\Command` (sergeymakinen)
+- Enh #15416: Added default roles dynamic definition support via closure for `yii\rbac\BaseManager` (deltacube)
```

Comment:
15422. The number you've used is not related.

Reply:
Oops, sorry. Just fixed